Simulator/Core/Simulator.cpp

```diff
@@ -288,6 +299,8 @@ BGFLOAT *Simulator::getPSummationMap() const { return pSummationMap_; }
 
 long Simulator::getSeed() const { return seed_; }
 
+long Simulator::getRNGSeed() const { return rngSeed_; }
```

Comment:
And we should make these member variables' and accessors' names clearer.

Reply:
I might get rid of `getSeed()` and `seed_` , now that it's being seeded right in `loadParameters()`. Unless there's ever a reason to re-seed.

Reply:
Renamed for clarity:
`getSeed()` -> `getNoiseRngSeed()`
`getRNGSeed()` -> `getInitRngSeed()`
`seed_` -> `noiseRngSeed_`
`rngSeed_` -> `initRngSeed_`